common/mixnet-contract/src/msg.rs

```diff
@@ -107,6 +114,10 @@ pub enum QueryMsg {
         address: Addr,
     },
     LayerDistribution {},
+    GetRewardPool {},
```

Comment:
Would it perhaps make sense to bundle all of those parameters together into a single struct (and thus single Query)?

Reply:
I wouldn't, this way each query does one thing, makes it easier to reason about them, at least for me :)